man/jackd.0

```diff
@@ -39,6 +39,12 @@ These do not include \fIbackend\-parameters\fR, which are listed using
 the \fB\-\-help\fR option for each specific backend.  Examples below
 show how to list them.
 .TP
+\fB\-a, \-\-autoconnect\fR \fImodechar\fR
+Select how to handle self connect requests. They can be ignore or fail, 
+on all port or just external ones. Use the \fB\-\-help\fR option to know 
+your system specific options. The default is to not restrict self connect 
+requests.".
```

Comment:
The last quote and dot on this line seem to be a typo. can you confirm?
I see nothing starting the quote

Reply:
Indeed, I opened the page in manedit and http://roperzh.github.io/grapse/ > to test.
The ". can be safely removed.   It seems that I cannot change the file anymore (must have deleted my repo.)  
Do you want me to create another pull request ?  
(Glad to see the project coming back to life by the way)